mobify/platform-scaffold
# Adds loader.js support for AMP pages for the new V8 tag

**JIRA**: https://mobify.atlassian.net/browse/AMP-103

## Reviewers
@kschmidtdev @olibrook 

## Changes
- Adds new AMP urls constant file for merlins amp routes
- Adds logic to loader.js to include amp links when appropriate

## Todos

## How to test-drive this PR
- Have the AMP-Validator chrome plugin installed
- `cd web/ && npm i && npm run dev`
- Set mobile emulation and preview Merlins and browse to the paths which have an amp page, i.e. `/potions, /books, etc`
- Verify that the validator plugin turns blue and that it reroutes you to the correct amp pages when you click on it
